### DISCUSSION TITLE
I am rolling back to 0.13.2. What happens next?

### DISCUSSION THREAD

**Discussion by f1d094** [blakeblackshear/frigate#13171] [Ask A Question] [answered]
### What is your question:

I will post my reasons in a new thread once I have compiled and organized them in a useful way. For now, let it suffice to say that 0.14 doesn't work for us and I am going to rollback to 0.13.2.

I saved my container, configs, and my pre-upgrade .db file...but I entirely forgot to make a filesystem snapshot of the media directory and cannot roll that back. We have been running on 0.14 for about 10 days or so and have several thousand more recordings/events.

What happens with the new events if I roll-back to my prior container, frigate.db, and config.yml? Should I delete all recording dirs since upgrade first or can I just leave them in place? Or...something else?

Please advise.

**Comment by NickM-27** [comment, marked as answer]
If you are going back to an older DB then the recordings made on 0.14 need to be deleted otherwise they will sit there indefinitely
